ksqldb-rest-client/src/test/java/io/confluent/ksql/rest/client/KsqlClientTest.java

```diff
@@ -739,8 +739,8 @@ public void shouldHandleArbitraryErrorsOnGetRequests() {
     List<StreamedRow> expectedResponse = new ArrayList<>();
     for (int i = 0; i < numRows; i++) {
       GenericRow row = GenericRow.genericRow("foo", 123, true);
-      StreamedRow sr = StreamedRow.pushRow(row);
-      expectedResponse.add(sr);
+//      StreamedRow sr = StreamedRow.pushRow(row);
+//      expectedResponse.add(sr);
```

Comment:
What's up with this?

Reply:
No idea.  Not sure why I would do such a thing.  Undone!